src/App.js

```diff
@@ -8,11 +12,23 @@ class App extends React.Component {
   }
 
   render() {
+    const mainStyle = { color: 'grey', fontSize: '20px', margin: '20px' };
     return (
-      <div className="App">
-        <div className="main-holder">
-          <Calculator />
-        </div>
+      <div className="App" style={mainStyle}>
+        <Router>
+          <NavBar />
+          <div className="main-holder">
+            <Routes>
+              <Route path="Calculator" element={<Calculator />} />
+            </Routes>
+            <Routes>
+              <Route path="Home" element={<Home />} />
+            </Routes>
```

Comment:
- [ OPTIONAL ] Your website root path is an empty page with just the navbar. It would be much better to display the `Home` page by default when the app is launched. You can achieve this by adding the `exact` route prop and using the `'/' ` for the Home page.
Hint: 
```jsx
<Route exact path="/">
   <Home />
</Route>
```